wordpress-mobile/WordPress-Android
# Issue/move whats new out of about category

This PR fixes a crash in App Settings related to introduction of unified about screen.

in #15551 FF check to remove About category from app settings was introduced. Unified About screen FF got turned ON now, so the About category is removed. Unintentionally, What's New preference was removed as well, which causes a crash when visiting App Settings with active feature announcement.

This PR moves the What's New preference to the parent screen.

cc @renanferrari @ravishanker 


To test:
- Set build flavor to jalapeno release and set app verstion to pr-15761-build-117882 
- Login to the app with a8c account.
- Navigate to App Settings, and confirm that "Wats New" category is visible and tapping on it shows announcement.

## Regression Notes
1. Potential unintended areas of impact
- "Wats New" might not be visible.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual testing.

3. What automated tests I added (or what prevented me from doing so)
- Legacy preference screen is not very testable.

PR submission checklist:
